src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/builders/AssertionHelper.java

```diff
@@ -59,108 +67,112 @@ public static EncryptedAssertion anEidasEncryptedAssertion(String requestId,
                                                 .build())
                                         .build())
                                 .build())
-                .withIssuer(
-                        anIssuer()
-                                .withIssuerId(issuerId)
-                                .build())
+                .withIssuer(anIssuer().withIssuerId(issuerId).build())
                 .addAttributeStatement(attributeStatement)
                 .addAuthnStatement(anEidasAuthnStatement().build())
                 .withSignature(assertionSignature)
                 .withConditions(aConditionsForEidas())
-                .buildWithEncrypterCredential(
-                        new TestCredentialFactory(
-                                TEST_RP_PUBLIC_ENCRYPTION_CERT,
-                                TEST_RP_PRIVATE_ENCRYPTION_KEY
-                        ).getEncryptingCredential()
-                );
+                .buildWithEncrypterCredential(target);
     }
 
-
-
-    public static EncryptedAssertion anEidasEncryptedAssertionWithInvalidSignature(String assertionIssuerId) {
```

Comment:
Don't we need this?

Reply:
Superceded by `aValidEidasResponseSignedByStubCountryWithAnInvalidSignatureOnTheAssertion`